Lib/ctypes/test/test_values.py

```diff
@@ -77,13 +77,154 @@ class struct_frozen(Structure):
                 bootstrap_seen.append(entry.name)
                 self.assertTrue(entry.size,
                     "{!r} was reported as having no size".format(entry.name))
-                continue
             items.append((entry.name.decode("ascii"), entry.size))
 
-        expected = [("__hello__", 164),
-                    ("__phello__", -164),
-                    ("__phello__.spam", 164),
-                    ]
+        expected = [
```

Comment:
I fixed this by comparing with the list of frozen modules from `imp._frozen_module_names()`, instead of generating the "expected" list in Tools/scripts/freeze_modules.py.